tcms/telemetry/templates/telemetry/testing-breakdown.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+{% load attachments_tags %}
+
+{% block title %}{% trans "Testing Breakdown" %}{% endblock %}
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+
+        <h1 class="no-test-cases-heading" style="visibility: hidden;">{% trans "No test cases for this product" %}</h1>
+
+        <div class="col-md-3">
```

Comment:
You are still missing HTML. These divs are inside of form-group, which is inside of form.

All of the texts below should be in label tags. If you can't get it right copy testcase/search.html and remove most of it and see how it looks. The current placement of widgets is nowhere good enough compared to the search templates.

They must have the same look and feel.